# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: js/src/tests/non262/fields/access.js
# Commit: d96f98f974e0
# Full Hash: d96f98f974e087224e0a1ef9be67e2fc63397d8b
# Author: Ashley Hauck <khyperia@mozilla.com>
# Date: 2019-03-21 04:34:45
# Regressor Bug: 1535166
# File Overlap Count: 1
# Description:
#   Bug 1535166 - Implement computed field names. r=jorendorff
#   
#   Differential Revision: https://phabricator.services.mozilla.com/D23408
# ==============================================================================

diff -r c786b72a9d8e -r d96f98f974e0 js/src/tests/non262/fields/access.js
--- a/js/src/tests/non262/fields/access.js	Wed Mar 20 15:50:30 2019 +0000
+++ /dev/null	Thu Jan 01 00:00:00 1970 +0000
@@ -1,19 +0,0 @@
-// * * * THIS TEST IS DISABLED - Fields are not fully implemented yet
-
-class C {
-    x = 5;
-}
-
-c = new C();
-
-reportCompare(c.x, undefined); // TODO
-//reportCompare(c.x, 5);
-
-class D {
-    #y = 5;
-}
-
-d = new D();
-
-reportCompare(d.#y, undefined); // TODO
-//reportCompare(d.#y, 5);